tanner/emulators/xxe_injection.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import logging
+
+from tanner.config import TannerConfig
+from tanner.utils.php_sandbox_helper import PHPSandboxHelper
+from tanner.utils import patterns
+
+
+class XXEInjection:
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.xxe_injection')
+        self.helper = PHPSandboxHelper(self._loop)
+
+    async def get_injection_result(self, code):
+
+        vul_code = '''<?php
+                        libxml_disable_entity_loader (false);
+                        $xml = \'%s\';
+                        $dom = new DOMDocument();
+                        $dom->loadXML($xml, LIBXML_NOENT | LIBXML_DTDLOAD);
+                        $creds = simplexml_import_dom($dom);
+
+                        $user = $creds->user;
```

Comment:
Here attacker needs to guess the name of the variable `<creds><user>&xxe;</user></creds>`. Instead
```suggestion
                        $data = simplexml_import_dom($dom);
                        echo $data;
```
would be better.

Reply:
To make it more intuitive we were emulating a submit form. So this can be exposed there.
Still should I change?

Reply:
I think this should be left for honeypot user and we should focus on making it work as it is. You can add information in the docs about how and what it can emulate.

Reply:
Alright.